src/components/search/search.pw.tsx

```diff
@@ -568,6 +577,24 @@ test.describe("Functionality tests for Search component", () => {
     });
 
     await expect(searchDefaultInputElement).toHaveValue("");
+    await expect(searchDefaultInputElement).toBeFocused();
+  });
+
+  test("should clear a Search input after enter key pressed and cross icon is focused", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<SearchComponent />);
+
+    const searchDefaultInputElement = searchDefaultInput(page);
+    await searchDefaultInputElement.clear();
+    await searchDefaultInputElement.type(testDataStandard);
```

Comment:
**suggestion**: what do you think about replacing the `type` action with `fill`? It seems [in Playwright v1.38 the `type` action has been deprecated](https://playwright.dev/docs/release-notes#version-138).

```suggestion
    await searchDefaultInputElement.fill(testDataStandard);
```